File: src/main/java/com/rkroom/blog/repository/TagRepository.java
package com.rkroom.blog.repository;

import com.rkroom.blog.entity.Tags;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface TagRepository extends JpaRepository<Tags,Integer> {
    List<Tags> findAll();
}

File: src/main/java/com/rkroom/blog/repository/TagRepository.kt
package com.rkroom.blog.repository

import com.rkroom.blog.entity.Tags
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface TagRepository : JpaRepository<Tags?, Int?> {
    override fun findAll(): List<Tags?>
}